pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
Normally the test suite run by maven utilizes 4 processes running concurrently, with each one utilizing 2GB, so computers with 16GB RAM (such as my laptop) can run the test suite. If you need more memory than 2GB per process, perhaps you can make it configurable.